package.json

```diff
@@ -44,7 +44,10 @@
     "moment-timezone": "^0.5.26",
     "node-sass": "^4.13.1",
     "portal-gun": "git://github.com/claydotio/portal-gun.git",
+    "promise-retry": "^1.1.1",
+    "queued-socket.io": "^1.0.5",
     "sass-loader": "^7.1.0",
+    "socket.io-client": "^2.3.0",
```

Comment:
Will `queued-socket.io` use this version of `socket.io-client`?

Reply:
The `queued-socket.io` packages specifies its peer dependency on `socket.io-client` as `^2.0.0`, which includes version `2.3.0`. There are no changes between `2.0.0` and `2.3.0` that would break the functioning of `queued-socket.io`.

Reply:
Okay cool. We seem to have had issues where sessions are stuck in an infinite loop of reconnect attempts, and I read that this can be caused by mismatched versions of socket.io on client & server

Reply:
(we have a few sessions that have hundreds and hundreds of "REJOINED SESSION" user actions)